dataParser/.gitignore

```diff
@@ -133,15 +133,6 @@ __pycache__
 
 __pycache__
 
-vendor/json-api/*.json
-vendor/json-api/en/*.json
-vendor/json-api/ru/*.json
-vendor/json-api/ko/*.json
-vendor/json-api/cmn-Hant/*.json
-vendor/json-api/ja/*.json
-vendor/json-api/de/*.json
-vendor/json-api/es/*.json
-vendor/client/tables/*.json
```

Comment:
NB: These were all duplicated in this file, removed these ones and left the others - no fallout here from this change.